src/NServiceBus.AcceptanceTests/ConfigureEndpointLearningPersistence.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using NServiceBus;
+using NServiceBus.AcceptanceTesting.Support;
+using NServiceBus.Persistence;
+
+public class ConfigureEndpointLearningPersistence : IConfigureEndpointTestExecution
+{
+    public Task Configure(string endpointName, EndpointConfiguration configuration, RunSettings settings, PublisherMetadata publisherMetadata)
+    {
+        storageDir = Path.Combine(@"c:\temp", Guid.NewGuid().ToString().Replace("-", "")); //can't use bindir since that will be to long on the build agents
```

Comment:
did a quick test and something to note here is that %TEMP% depends on the user, so running this on a regular user would yield something like this:
`C:\Users\<Username>\AppData\Local\Temp` which I think is too long again.
the systems value btw. isn't `C:\Temp` but `C:\Windows\Temp`.

so should we stay with the hardcoded value?